lib/Cleantalk/ApbctWP/Firewall/SFW.php

```diff
@@ -661,6 +661,14 @@ public static function directUpdate($db, $db__table__data, $blacklists)
      */
     public static function updateWriteToDb($db, $db__table__data, $file_url = null)
     {
+        $table_exist = $db->fetchAll(
+            'SHOW TABLES LIKE "' . $db__table__data . '";'
```

Comment:
Use `\Cleantalk\Common\DB::isTableExists()` instead of direct query

Reply:
fixed https://github.com/CleanTalk/wordpress-antispam/pull/290/commits/836df1802a2ff8c377c5e7c07c723187b36cc9af